src/bundle/Resources/views/content/locationview.html.twig

```diff
@@ -73,7 +73,6 @@
                             'system_urls_pagination_params': system_urls_pagination_params,
                             'roles_pagination_params': roles_pagination_params,
                             'policies_pagination_params': policies_pagination_params,
-                            'is_location_visible': is_location_visible,
```

Comment:
Technically this is BC break, so let leave this variable (with deprecation node) and pass here location.invisible